dbms/tests/queries/0_stateless/00965_pocopatch_logs_level_bugfix.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. $CURDIR/../shell_config.sh
+
+cp /dev/null 00965_logs_level_bugfix.tmp
```

Comment:
Why `cp /dev/null`?

Reply:
This line creates .tmp file if it doesn't exist + clears it otherwise

Reply:
```
> 00965_logs_level_bugfix.tmp
```